addons/mail/wizard/template_reset_wizard.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, _
+
+
+class TemplateResetWizard(models.TransientModel):
+    _name = 'template.reset.wizard'
```

Comment:
``mail.template.reset`` maybe to keep the mail.template naming ? Would also require to update files / views / ... ^^ and wizard suffix is not really necessary